app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    cmd, source_path, destination_path = command.split()
```

Comment:
Before you must check that command.split() length == 3

Reply:
also check if `cmd == "mv"`

Reply:
> also check if `cmd == "mv"`

string 8,9:
    if cmd != "mv":
        raise ValueError('Invalid command. Only "mv" command is supported.')

Reply:
> Before you must check that command.split() length == 3

Why? If the length of command.split() is not equal to 3, a ValueError will automatically be raised: "not enough values to unpack (expected 3, got 2 or 1, 0...)" or "ValueError: too many values to unpack", and further execution of the code will be impossible.

Reply:
Okay, for you is it an understandable exception? As for me no, we need provide an informative description

Reply:
Validation must be before unpacking